pkg/state/state_hostdevice_network_test.go

```diff
@@ -29,6 +31,14 @@ import (
 	"github.com/Mellanox/network-operator/pkg/utils"
 )
 
+func checkResourceNameAnnotation(obj *unstructured.Unstructured) {
+	annotations := obj.Object["metadata"].(map[string]interface{})["annotations"].(map[string]interface{})
+	resourceName := annotations["k8s.v1.cni.cncf.io/resourceName"].(string)
+
+	Expect(strings.HasPrefix(resourceName, resourceNamePrefix)).To(Equal(true))
+	Expect(strings.Count(resourceName, resourceNamePrefix)).To(Equal(1))
```

Comment:
nit: Expect(strings.Count(resourceName, resourceNamePrefix)).To(Equal(1)) > Expect(resourceName).To(HavePrefix(resourceNamePrefix)) ?